docs/integrations/prefect-azure/index.mdx

```diff
@@ -9,57 +9,39 @@ For example, you can retrieve secrets, read and write Blob Storage objects, and
 
 ### Prerequisites
 
-- [Prefect installed](3.0/get-started/install) in a virtual environment.
 - An [Azure account](https://azure.microsoft.com/) and the necessary permissions to access desired services.
 
 ### Install prefect-azure
 
-Install or update to the latest version of the prefect-azure library and dependencies.
+The following command will install a version of prefect-azure compatible with your installed version of Prefect. 
```

Comment:
```suggestion
The following command will install a version of `prefect-azure` compatible with your installed version of `prefect`. 
```

Reply:
Cool. Pushing changes to all integrations docs that reflect these changes.